simtools/camera_efficiency.py

```diff
@@ -262,41 +266,54 @@ def analyze(self, export=True, force=False):
         self._has_results = True
 
         print("\33[40;37;1m")
-        self._logger.info(f"Spectrum weighted reflectivity: {self.calc_reflectivity():.4f}")
-        self._logger.info(
-            f"Camera nominal efficiency with gaps (B-TEL-1170): {self.calc_camera_efficiency():.4f}"
-        )
-        self._logger.info(
-            "Telescope total efficiency"
-            f" with gaps (was A-PERF-2020): {self.calc_tel_efficiency():.4f}"
-        )
-        self._logger.info(
-            (
-                "Telescope total Cherenkov light efficiency / sqrt(total NSB efficency) "
-                "(A-PERF-2025/B-TEL-0090): "
-                f"{self.calc_tot_efficiency(self.calc_tel_efficiency()):.4f}"
-            )
-        )
-        self._logger.info(
-            "Expected NSB pixel rate for the reference NSB: "
-            f"{self.calc_nsb_rate():.4f} [p.e./ns]"
-        )
+        self._logger.info(f"\n{self.results_summary()}")
         print("\033[0m")
 
         if export:
             self.export_results()
 
     # END of analyze
 
+    def results_summary(self):
+        """
+        Print a summary of the results.
+        Include a header for the zenith/azimuth settings and the NSB spectrum file which was used.
+        The summary includes the various CTAO requirements and the final expected NSB pixel rate.
+        """
+        summary = (
```

Comment:
I was considering the option of saving the time stamp as well in the summary. Sometimes it is helpful to know when a report/summary was generated. Would you also consider printing out the version of the tool (testeff or sim_telarray) and version of simtools?

Reply:
Those are all provenance information which should be collected "automatically" by the tools running the application (like CWL). I must say also that this summary was written only for something very specific. I think this function will change later to write an ecsv file with the results and additional metadata.